components/PersonalFinance/Slider/Loans.js

```diff
@@ -1,12 +1,12 @@
 import { formatCurrency } from '../../../utils/formatCurrency';
 
-export function Loans({ balance }) {
+export function Loans({ balance,institutionId }) {
   return (
     <div className="border-2 shadow-md h-44 sm:h-64 bg-[#fbfbfb] rounded-2xl border-[#e0eaff] sm:bg-[#F5F7F8] sm:min-w-max">
       <div className="mt-4 mb-2 ml-7 mr-9 sm:mb-5">
         <div className="flex items-center">
           <div>
-            <img className="w-7 h-7" src="/slider/bank-of-melbourne.svg" alt="Bank of Melbourne" />
+            <img className="w-7 h-7" src={institutionId != null?`/bank_icons/${institutionId}.svg`:'/bank-illustration.svg'} />
```

Comment:
We should use loading @bulentozdil If there is not any logo for the institution then we can use bank-illustration.svg as default logo